{"primaryContentSections":[{"declarations":[{"platforms":["iOS"],"languages":["swift"],"tokens":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"underlying","kind":"identifier"},{"kind":"text","text":"("},{"text":"error","kind":"externalParam"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP","text":"Error"},{"kind":"text","text":")"}]}],"kind":"declarations"}],"kind":"symbol","identifier":{"interfaceLanguage":"swift","url":"doc:\/\/Networking\/documentation\/Networking\/NetworkError\/underlying(error:)"},"hierarchy":{"paths":[["doc:\/\/Networking\/documentation\/Networking","doc:\/\/Networking\/documentation\/Networking\/NetworkError"]]},"schemaVersion":{"patch":0,"major":0,"minor":3},"variants":[{"paths":["\/documentation\/networking\/networkerror\/underlying(error:)"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"externalID":"s:10Networking12NetworkErrorO10underlyingyACs0C0_p_tcACmF","title":"NetworkError.underlying(error:)","symbolKind":"case","role":"symbol","modules":[{"name":"Networking"}],"roleHeading":"Case","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"underlying"},{"kind":"text","text":"("},{"kind":"externalParam","text":"error"},{"kind":"text","text":": "},{"text":"Error","kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":")"}]},"abstract":[{"text":"An indication of an underlaying network layer error.","type":"text"}],"sections":[],"references":{"doc://Networking/documentation/Networking/Response":{"kind":"symbol","role":"symbol","navigatorTitle":[{"text":"Response","kind":"identifier"}],"title":"Response","identifier":"doc:\/\/Networking\/documentation\/Networking\/Response","type":"topic","fragments":[{"kind":"keyword","text":"typealias"},{"kind":"text","text":" "},{"kind":"identifier","text":"Response"}],"url":"\/documentation\/networking\/response","abstract":[{"type":"text","text":"A default "},{"code":"URLSession","type":"codeVoice"},{"type":"text","text":" response tuple."}]},"doc://Networking/documentation/Networking":{"type":"topic","abstract":[],"url":"\/documentation\/networking","kind":"symbol","role":"collection","identifier":"doc:\/\/Networking\/documentation\/Networking","title":"Networking"},"doc://Networking/documentation/Networking/NetworkError/underlying(error:)":{"abstract":[{"type":"text","text":"An indication of an underlaying network layer error."}],"url":"\/documentation\/networking\/networkerror\/underlying(error:)","identifier":"doc:\/\/Networking\/documentation\/Networking\/NetworkError\/underlying(error:)","title":"NetworkError.underlying(error:)","kind":"symbol","type":"topic","role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"underlying"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"error"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"text":")","kind":"text"}]},"doc://Networking/documentation/Networking/NetworkError":{"abstract":[{"type":"text","text":"An error that occurs during "},{"identifier":"doc:\/\/Networking\/documentation\/Networking\/Response","isActive":true,"type":"reference"},{"text":" processing or an underlaying ","type":"text"},{"isActive":true,"identifier":"doc:\/\/Networking\/documentation\/Networking","type":"reference"},{"type":"text","text":" error."}],"url":"\/documentation\/networking\/networkerror","identifier":"doc:\/\/Networking\/documentation\/Networking\/NetworkError","title":"NetworkError","type":"topic","kind":"symbol","role":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"kind":"identifier","text":"NetworkError"}],"navigatorTitle":[{"text":"NetworkError","kind":"identifier"}]}}}